compiler/rustc_codegen_ssa/src/back/rpath/tests.rs

```diff
@@ -1,13 +1,6 @@
-use std::ffi::OsString;
 use std::path::{Path, PathBuf};
 
-use super::{RPathConfig, get_rpath_relative_to_output, minimize_rpaths, rpaths_to_flags};
-
-#[test]
-fn test_rpaths_to_flags() {
-    let flags = rpaths_to_flags(vec!["path1".into(), "path2".into()]);
-    assert_eq!(flags, ["-Wl,-rpath,path1", "-Wl,-rpath,path2"]);
-}
+use super::{RPathConfig, get_rpath_relative_to_output, minimize_rpaths};
```

Comment:
```suggestion
use super::*;
```
This is the standard for test modules.